prow/plugins/projectmanager/projectmanager_test.go

```diff
@@ -747,15 +747,15 @@ func checkCards(expectedColumnCards, projectColumnCards map[int][]github.Project
 				}
 			}
 			if !found {
-				return fmt.Errorf("Unable to find project card: %v under column: %d", card, columnID)
+				return fmt.Errorf("unable to find project card: %v under column: %d", card, columnID)
 			}
 		}
 	}
 	return nil
 }
 
 func TestHelpProvider(t *testing.T) {
-	var i int = 0
+	var i = 0
```

Comment:
nit: `var i int` is IMO a bit more idiomatic

Reply:
https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style is the source I cite for this

Reply:
Fixed